packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -259,25 +266,16 @@ class DatePicker extends React.Component {
       this.props.onSelect(event, selectedDate.format(DateUtil.ISO_EXTENDED_DATE_FORMAT));
     }
 
-    if (!this.props.disableButtonFocusOnClose) {
-      // Allows time for focus-trap to release focus on the picker before returning focus to the calendar button.
-      setTimeout(() => {
-        /*
-         * Make sure the reference to calendarButton still exists before calling focus because it is possible that it is now
-         * nullified after the 100 ms timeout due to a force remount of this component with a new `key` prop value.
-         * Reference https://github.com/cerner/terra-framework/issues/1086
-         */
-        if (this.calendarButton) {
-          this.calendarButton.focus();
-        }
-      }, 100);
```

Comment:
Why is this removed?

Reply:
SInce this piece of code is required multple times, I moved this piece of code to `focusCalendarButton` method.